src/main/java/mate/academy/dao/CinemaHallDao.java

```diff
@@ -10,4 +10,8 @@ public interface CinemaHallDao {
     Optional<CinemaHall> get(Long id);
 
     List<CinemaHall> getAll();
+
+    boolean update(CinemaHall cinemaHall);
```

Comment:
Consider changing the return type of the `update` method to `CinemaHall` or `void` to better reflect its operation. Returning a boolean might not provide enough information about the update process.